crates/turborepo/Cargo.toml

```diff
@@ -36,6 +36,7 @@ clap_complete = { workspace = true }
 command-group = { version = "2.0.1", features = ["with-tokio"] }
 dunce = { workspace = true }
 human-panic = "1.2.1"
+miette.workspace = true
```

Comment:
We should get rid of `go-binary` and `go-daemon` features (Line not relevant, just file)